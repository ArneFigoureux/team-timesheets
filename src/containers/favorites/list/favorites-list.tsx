import React from "react";
import { Doc } from "firestorable";
import { IRegistration } from "../../../../common";
import { RegistrationLines } from "../../registrations/lines";

export const FavoritesList = ({
    favorites,
}: {
    favorites: Doc<IRegistration>[],
}) => {
    return (
        <RegistrationLines
            readOnly={true}
            registrationClick={() => { }}
            registrations={favorites}
        ></RegistrationLines>
    );
}
