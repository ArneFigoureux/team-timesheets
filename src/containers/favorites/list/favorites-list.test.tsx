import React from "react";
import { render } from "@testing-library/react";
import { FavoritesList } from "./favorites-list";

describe("FavoritesList", () => {
    it("should render without favorites", () => {
        const { asFragment } = render(
            <FavoritesList
                favorites={[]}
            />
        );

        expect(asFragment()).toMatchSnapshot();
    });

    // it("should render favorites", () => {
    //     const { asFragment } = render(
    //             <FavoritesList
    //                 favorites={[
    //                     {
    //                         data: {
    //                             groupId: "group-1",
    //                             userId: "user-1",
    //                             client: "client-1",
    //                             description: "Favorite desc 1",
    //                             project: "project-1",
    //                             task: "task-1",
    //                             time: 3
    //                         },
    //                         id: "favorite-1",
    //                     } as unknown as Doc<IFavoriteRegistration>
    //                 ]}
    //             />
    //     );

    //     expect(asFragment()).toMatchSnapshot();
    // });
});