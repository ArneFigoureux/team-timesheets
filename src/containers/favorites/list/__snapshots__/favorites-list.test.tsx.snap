// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoritesList should render without favorites 1`] = `<DocumentFragment />`;
